braid/communication.c

```diff
@@ -108,17 +108,16 @@ _braid_CommSendInit(braid_Core           core,
       /* Store the receiver rank in the status */
       _braid_StatusElt(bstatus, send_recv_rank) = proc;
 
-      /* Note that bufpack may return a size smaller than bufsize */ 
-      _braid_StatusElt(bstatus, size_buffer) = size;
```

Comment:
Why is this line being removed?  Don't we want to tell the user what the current buffer size is through bstatus?